docs/source/getting_started_guides/self_driving/01_loading_datasets.ipynb

```
@@ -0,0 +1,506 @@
+    "    sample[\"location\"] = fo.GeoLocation(\n",
+    "        point = [lon,lat]\n",
+    "    )\n",
+    "    for box in boxes:\n",
+    "        if box_in_image(box,camera_intrinsic,shape,vis_level=BoxVisibility.ALL):\n",
+    "            c = np.array(nusc.colormap[box.name]) / 255.0\n",
+    "            corners = view_points(box.corners(), camera_intrinsic, normalize=True)[:2, :]\n",
+    "            front = [(corners[0][0]/width,corners[1][0]/height),\n",
+    "                    (corners[0][1]/width,corners[1][1]/height),\n",
+    "                    (corners[0][2]/width,corners[1][2]/height),\n",
+    "                    (corners[0][3]/width,corners[1][3]/height),]\n",
+    "            back =  [(corners[0][4]/width,corners[1][4]/height),\n",
```

Ruff (0.12.2)

Redefinition of unused `BoxVisibility` from line 62

Remove definition: `BoxVisibility`

(F811)